tests/test_meltano_utils.py

```diff
@@ -406,3 +406,71 @@ def test_sdk_about_parsing_skip_default_dates():
             "kind": "date_iso8601",
         }
     ]
+
+
+@pytest.mark.parametrize(
+    "input,expected",
+    [
+        # Official
+        [["meltanolabs", True, 0, "low"], "gold"],
+        [["meltanolabs", True, 0, "medium"], "gold"],
+        [["meltanolabs", True, 0, "high"], "gold"],
+        [["meltanolabs", True, 1, "low"], "gold"],
+        [["meltanolabs", True, 1, "medium"], "gold"],
+        [["meltanolabs", True, 1, "high"], "gold"],
+        [["meltanolabs", True, 6, "low"], "gold"],
+        [["meltanolabs", True, 6, "medium"], "gold"],
+        [["meltanolabs", True, 6, "high"], "gold"],
+        [["meltanolabs", False, 0, "low"], "gold"],
+        [["meltanolabs", False, 0, "medium"], "gold"],
+        [["meltanolabs", False, 0, "high"], "gold"],
+        [["meltanolabs", False, 1, "low"], "gold"],
+        [["meltanolabs", False, 1, "medium"], "gold"],
+        [["meltanolabs", False, 1, "high"], "gold"],
+        [["meltanolabs", False, 6, "low"], "gold"],
+        [["meltanolabs", False, 6, "medium"], "gold"],
+        [["meltanolabs", False, 6, "high"], "gold"],
```

Comment:
@pnadolny13 I would say non-sdk based connectors should not be Gold. Perhaps max is silver and it's a clear signal we need to remake it on the SDK.

The only exception would be a high-usage/responsiveness connector - but it depends on what the connector actually is.

Reply:
@tayloramurphy I agree with that, here are the meltano/meltanolabs variants that arent SDK based. I think silver/bronze are accurate for all of these:

- tap-salesforce - meltanolabs
- tap-sftp - meltanolabs
- tap-gitlab - meltanolabs
- target-athena - meltanolabs
- target-sqlite - meltanolabs
- tap-fastly - meltano (GitLab archived)

Reply:
@pnadolny13 yep - that list feels like a silver/bronze list!